### ISSUE TITLE
Define who's the recipient of a shielding transaction

### ISSUE THREAD

**Issue by pacu** [zcash/ZcashLightClientKit#521]
Function Shield funds calls `initSpend`
````
public func shieldFunds(
        transparentAccountPrivateKey: TransparentAccountPrivKey,
        memo: String?,
        from accountIndex: Int,
        resultBlock: @escaping (Result<PendingTransactionEntity, Error>) -> Void
    ) 
````
Which requires a recipient Address
`let shieldingSpend = try transactionManager.initSpend(zatoshi: tBalance.verified, toAddress: zAddr, memo: memo, from: 0)`

There's no such string encoded recipient address for shielding transactions, since Internal Addresses are not string encoded since they are not intended to be displayed to users.

**Comment by nuttycom** [comment]
The recipient of the shielding operation is the account index that's being provided here. The wallet front-end might want to do something where it allows users to provide friendly names for accounts, so that it's not just `wallet internal account 1` or whatever.

**Comment by pacu** [comment]
closed in #567
